### DISCUSSION TITLE
[Feature Request] Sync from Multiple Trakt Lists

### DISCUSSION THREAD

**Discussion by neo-neo1** [pymedusa/Medusa#10074] [Ideas & Feature requests]
If you use Trakt chances are you use multiple lists. For example on Radar users often add multiple community maintained Trakt lists such as "Top monthly torrented movies", "Biggest weekly box office releases", and their own personal Trakt list. In addition, multiple Trakt lists is a simple way add remote access for multiple users - to add their requested shows to Medusa.

You can even use mdblist.com to make custom lists. As an example: source from "Top monthly viewed shows" list then narrow down by Sci-Fi. Then narrow down by space themed show, or specific actors, and etc.. . Very powerful tool with tons of variables to custom tailor lists to your liking.

**Comment by p0psicles** [comment]
Yeah I understand sounds cool. But this would make the trakt integration exponentially more complex. As you would also want to have different configurations per trakt list. And what if list x adds a show to medusa, but list y removes it again. Would create oppurtunities for a lot of issues.

So this is a high risk, low reward kind of feature.

**Reply by neo-neo1** [reply]
> Yeah I understand sounds cool. But this would make the trakt integration exponentially more complex. As you would also want to have different configurations per trakt list. And what if list x adds a show to medusa, but list y removes it again. Would create oppurtunities for a lot of issues.
> 
> So this is a high risk, low reward kind of feature.

My entire Trakt use case is 1-way syncing, as is for others as well. Only Trakt to Medusa syncs actually matter. If I could disable Medusa pushing its library to Trakt I would. This is how Radarr functions, no 2 way syncing. Medusa does more than it needs to which [ultimately causes issues  
](https://github.com/pymedusa/Medusa/issues/10073#issue-1051502458)
Using Trakt to actually track show watching is better left for Kodi, Plex, or whichever player the user uses. 

The one thing Medusa intrinsically does is download shows. And when paired with Trakt lists equates a powerful tool. I can’t make Kodi download shows but I can make it track shows.  And you don’t use Medusa to view shows.

**Reply by p0psicles** [reply]
I'm okay with adding the 1-way sync. I was even in the impression this was already possible. But at least it should
